newrelic/resource_newrelic_nrql_alert_condition.go

```diff
@@ -324,14 +325,15 @@ func resourceNewRelicNrqlAlertCondition() *schema.Resource {
 				Description:  "How long we wait after each data point arrives to make sure we've processed the whole batch. Use aggregationTimer with the EVENT_TIMER aggregation method.",
 				RequiredWith: []string{"aggregation_method"},
 				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
-					sinceValueExists := d.Get("nrql.0.since_value") != nil
+					_, sinceValueExists := d.GetOk("nrql.0.since_value")
 					if sinceValueExists {
 						return false
 					}
 					// If a value is not provided and the condition uses the default value, don't show a diff
 					oldInt, _ := strconv.ParseInt(old, 0, 8)
 					newInt, _ := strconv.ParseInt(new, 0, 8)
-					return oldInt == 120 && (newInt == 0) && d.Get("aggregation_method") == "EVENT_TIMER"
+					aggregationMethod := strings.ToLower(d.Get("aggregation_method").(string))
+					return oldInt == 60 && newInt == 0 && aggregationMethod == "event_timer"
```

Comment:
changed `120` to the default `60` as per [nrql_alert_condition docs](https://github.com/newrelic/terraform-provider-newrelic/blob/main/website/docs/r/nrql_alert_condition.html.markdown#argument-reference)

Reply:
👏